CONTRIBUTING.rst

```diff
@@ -0,0 +1,78 @@
+Contributing
+============
+
+``web-po`` is an open-source project and we'd love to accept your
+contributions. There are just a few small guidelines that you need to follow.
+
+How to report issues
+--------------------
+
+Please, create a new issue on this repository if you have:
+
+* a bug
+* a new feature
+* a change proposal
+
+Make sure to include all necessary details that could help other contributors
+to understand or to reproduce your use case.
+
+How to submit a patch
+---------------------
+
+* Fork the project
+* Create a new branch with your patch
+* Open a pull request to master
+* Ask for code reviews from other contributors
+
+Ideally, tests should keep passing and coverage should not decrease after a
+commit.
+
+How to release a new version
```

Comment:
This information is not for contributors, it is for maintainers.

Reply:
move to MAINTAINING.rst?

Reply:
yeah, or to wiki.

Reply:
moved to a new file